src/main/java/com/sap/cloudfoundry/client/facade/rest/CloudControllerRestClientImpl.java

```diff
@@ -374,10 +374,7 @@ private CreateApplicationRequest.Builder createApplicationRequestBuilder(String
     private BuildpackData createBuildpackData(Staging staging) {
         BuildpackData.Builder buildpackDataBuilder = BuildpackData.builder()
                                                                   .stack(staging.getStackName());
-        if (staging.getBuildpack() != null) {
-            buildpackDataBuilder.buildpack(staging.getBuildpack());
-        }
-        if (shouldUpdateWithV3Buildpacks(staging)) {
+        if (staging.getBuildpacks() != null) {
```

Comment:
Have you tested if a single buildpack is provided in the deployment descriptor? Does it get passed in "buildpacks"
- name: module
   parameters:
     buildpack: test